src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/LetterControllerITest.java

```diff
@@ -6791,18 +6789,6 @@ void verifyDataEnglish(PrintLetterDataResponseDto response,
             assertThat(response.getJurorNumber()).as("Expect juror number to be 5555555" + jurorPostfix)
                 .isEqualTo(JUROR_NUMBER + jurorPostfix);
 
-            assertThat(response.getNonAttendance())
```

Comment:
now that these have been removed, do we not need to check the response has the details in the list of attendances, e.g. for each day